__tests__/app.test.js

```diff
@@ -144,28 +144,88 @@ describe("GET: /api/articles/:article_id/comments", () => {
         });
       });
   });
-  test('GET:404 should send an appropriate status and error message when passed an id that is valid but does not exist', () => {
+  test("GET:404 should send an appropriate status and error message when passed an id that is valid but does not exist", () => {
     return request(app)
-    .get('/api/articles/999/comments')
-    .expect(404)
-    .then((response) => {
-      expect(response.body.msg).toBe('article does not exist');
-    });
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("article does not exist");
+      });
+  });
+  test("GET:200 should respond with an empty array if there are no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments).toEqual([]);
+      });
+  });
+  test("should respond with an appropriate status and error message when passed an invalid id", () => {
+    return request(app)
+      .get("/api/articles/abc/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request");
+      });
+  });
+});
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("201: should respond with an object with a single posted comment", () => {
+    const newComment = {
+      article_id: 1,
+      author: "butter_bridge",
+      body: "story of my life",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        console.log(response.body, "new comment");
+        expect(response.body.comments.article_id).toBe(1);
+        expect(response.body.comments.author).toBe("butter_bridge");
+        expect(response.body.comments.body).toBe("story of my life");
+      });
```

Comment:
Please add another happy path test where unnecessary properties are ignored. For example when posting with:
`{     
      author: "butter_bridge",
      body: "story of my life",
      extraProp: "extraValue"
    }`